### ISSUE TITLE
Show version string in TensorBoard UI

### ISSUE THREAD

**Issue by dandelionmane** [tensorflow/tensorboard#157]
In https://github.com/tensorflow/tensorboard/issues/150, user had an issue that was made more confusing by the fact that they were actually hitting a stale cache and getting an older version of TensorBoard.

We can make this clearer by presenting the version string in the TensorBoard UI. We could have the core plugin serve a route for this. If we want to avoid the caching issue, we should find a way to inject it into index.html; if we just make a new route served by the core plugin, then possibly the user would get a stale index, but the new route. (Alternatively, we could serve the version string from a route, but ensure it has the same caching policy as index.html? Not quite as robust, but quite possibly easier for us to maintain.)

Actually - I wonder if we could use a BUILD time dependency to get the version string into the UI? @jart @wchargin thoughts?

**Comment by jart** [comment]
Maybe have a genrule print `tensorboard.version.VERSION` to a file and html import it.

**Comment by wchargin** [comment]
Yep, agree that build time is best, and what @jart said makes sense to me.
